packages/components/Tabs/src/TabsItem.types.ts

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import { ViewProps } from 'react-native';
+import { IRenderData } from '@uifabricshared/foundation-composable';
+import type { IViewProps } from '@fluentui-react-native/adapters';
+import { IFocusable, IPressableState } from '@fluentui-react-native/interactive-hooks';
+import { ITextProps } from '@fluentui-react-native/text';
+import { IconProps } from '@fluentui-react-native/icon';
+import { IButtonProps, IButtonTokens } from '@fluentui-react-native/button';
+
+export const tabsItemName = 'TabsItem';
+
+export interface TabsItemInfo extends IPressableState {
+  /**
+   * TabsItem icon.
+   */
+  icon?: boolean;
+
+  /**
+   * TabsItem text.
+   */
+  headerText?: boolean;
+
+  /**
+   * Indicates if TabsItem is selected.
+   */
+  selected: boolean;
+
+  /**
+   * Key for tabItem
+   */
+  key: string;
+}
+
+export interface TabsItemState {
+  info: TabsItemInfo;
+}
+
+export interface TabsItemProps extends IButtonProps {
+  /*
+   ** The text string for the option
+   */
+  headerText?: string;
+
+  /*
+   ** The number for the TabsItem count
+   */
+  itemCount?: number;
+
+  /*
+   ** A unique key-identifier for each option
+   */
+  itemKey: string;
+
+  /*
+   ** Whether or not the tabs item is selectable
+   */
+  disabled?: boolean;
+
+  /*
+   ** An optional string for the Narrator to read for each TabsItem. If not provided, this will be set to the tabsItem's content
+   */
+  ariaLabel?: string;
+
```

Comment:
I think you should also add `ariaPosInSet` and `ariaSetSize` to TabItemProps so someone can define the n of m properties if they aren't generated (for instance, if TabsItems isn't a direct child of Tabs). You can find an example of the implementation in RadioButton: https://github.com/microsoft/fluentui-react-native/blob/master/packages/components/RadioGroup/src/RadioButton.tsx#L85

Reply:
Thanks for the reference, those props have been added.